### PR TITLE
Prepared for generic page layout

### PR DESCRIPTION
**Pull request by SociableSteve**
Prepares for #26

### CODE REVIEW

File: assets/style/bioconductor-v2.css [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-/* "site-container" is the single div containing every other element in the site.
-   it floats center, with no vertical margins, and provides the overall
-   width of the site. we provide both an id and a class name in order
-   to allow more flexible cascading: id rules are highly ranked while
-   class names play nicer with other rules. */
-
-#site-container {
-  margin: 0 auto; /* floats the design center */
-  width: 1600px; /* change the site design width here */
+.container {
```
Comment (MariamM110) [inline assets/style/bioconductor-v2.css:6]: Should the `site-container` class/id be removed at this point from the html default and course and just reference this container instead?

Comment (SociableSteve) [reply assets/style/bioconductor-v2.css:6]: Funny you should say that. I've just pushed that particular change up 😁

File: assets/style/bioconductor-v2.css [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
+  margin: 0 auto;
+  width: calc(100% - 3rem);
+  max-width: 1600px;
```
Comment (bp289) [inline assets/style/bioconductor-v2.css]: shouldnt the max-width be 1400px ? I thought nick changed it last week.

Comment (SociableSteve) [reply assets/style/bioconductor-v2.css]: Good catch. Just double checked with @nickzafiropulos and it should be 1400px. I've update this.

**Review by MariamM110 — DISMISSED** [review]
Approved for merging. @SociableSteve will pull any tidy up work into another ticket.

**Review by bp289 — DISMISSED** [review]

**Review by MariamM110 — DISMISSED** [review]
Approved for merging. @SociableSteve will pull any tidy up work into another ticket.

**Review by bp289 — APPROVED** [review]

**Review by MariamM110 — APPROVED** [review]
Approved for merging. @SociableSteve will pull any tidy up work into another ticket.
- Changes will be made to other pages

**Pull request merged**